frontend/rpm/template.go

```diff
@@ -499,6 +499,14 @@ func (w *specWrapper) Install() fmt.Stringer {
 		}
 	}
 
+	if w.Spec.Artifacts.Libexec != nil {
```

Comment:
can you do the same in `frontend/deb` and add a test in `test/` with the other artifact types?

Reply:
Yes

Reply:
But will I?

(the answer is still yes)

Reply:
done